packages/block-library/src/separator/style.scss

```diff
@@ -1,6 +1,6 @@
 .wp-block-separator {
-	border-top: 1px solid currentColor;
-	border-bottom: 1px solid currentColor;
+	border-top: 2px solid currentColor;
+	border-bottom: none;
```

Comment:
It would make sense to move this to line 7 where the other borders are unset, and update the comment to include "bottom"

Reply:
Thanks, done in https://github.com/WordPress/gutenberg/pull/55725/commits/33e1f732f456dd5a255e4b66fd5e4ad4c706b364